### PR TITLE
Adds branding on navigation

### PR DESCRIPTION
**Pull request by a2batic**
Before : 
screenshot from 2019-01-07 14-55-59

After : 
screenshot from 2019-01-07 14-55-15

Signed-off-by: Kanika Murarka <kmurarka@redhat.com>

### CODE REVIEW

File: src/pybind/mgr/dashboard/frontend/angular.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
             "polyfills": "src/polyfills.ts",
             "assets": [
               "src/assets",
-              "src/favicon.ico"
+              "src/favicon.ico",
+	      "src/branding/assets"
```
Comment (gnehapk) [inline src/pybind/mgr/dashboard/frontend/angular.json]: @a2batic Please fix spacing here.

File: src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
+<nav class="navbar navbar-default navbar-pf">
   <!-- Brand and toggle get grouped for better mobile display -->
 
   <div class="navbar-header tc_logo_component">
     <a class="navbar-brand"
        href="#">
-      <img src="assets/Ceph_Logo_Standard_RGB_White_120411_fa.png"
-           alt="Ceph">
+      <img src="/branding/assets/RHCSD-Masthead-Logo.svg"
+           width="100%"
```
Comment (gnehapk) [inline src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html]: @a2batic Please use css class for width.

File: src/pybind/mgr/dashboard/frontend/angular.json (hunk 1 above)
Comment (a2batic) [reply src/pybind/mgr/dashboard/frontend/angular.json]: ack

File: src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html (hunk 2 above)
Comment (a2batic) [reply src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html]: ack

**Comment by julienlim** [comment]
Based on screenshots, it looks good to me!

**Pull request merged**